tests/e2e/Readme.md

```diff
@@ -1,4 +1,4 @@
-# Azure IoT middleware E2E Test Framework
+# Azure IoT Middleware E2E Test Framework

```

Comment:
this was decided to be kept lowercase across docs. @wduraes can have ultimate say since he's the "master of the optics"

Reply:
Got it. I'll change it back for now.

@wduraes should we consider this is part of a title in this case?

Reply:
Decided offline this is a title so we'll have it capitalized here. Lowercase everywhere else.